mitype/app.py

```diff
@@ -4,6 +4,8 @@
 import os
 import sys
 import time
+import urllib.parse
```

Comment:
`six` isn't installed in python3 by default so should I add it as a dependency in `setup.py`?

Reply:
Nice catch. Let's try to avoid external dependencies.
Since the newline and the `#` characters are the only ones that need to be converted manually, a better approach could be just using `message = message.replace("\n", "%0D").replace("#", "%23")`. The rest of the characters are fine as is.